how-to-docs/README.md

```diff
@@ -16,3 +16,5 @@ This folder contains a collection of how-to guides for both Developers and DevOp
   - [Network Policies](network-policies.md)
   - [Pod Security Policies](pod-security-policies.md)
   - [Provisioned Volumes and Storage Classes](pv-and-storage-classes.md)
+  - [Autoscaling Pods](autoscaling.md)
```

Comment:
Is this doc missing in this PR?

Reply:
The doc already exists, just was never linked here so I added it in. :)